src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -224,7 +243,7 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
     async connectedCallback() {
         try {
             this.donationMatchingBehaviors = await getDonationMatchingValues();
-
+            await GeFormService.getFormTemplate();
```

Comment:
What is the reasoning for getting the form template here?  Something to do with initializing the field mappings?

Reply:
Yes, we need the mappings to be present for `appendToElement` on line 222 where we figure out the source field's API name.  Later on we use it to determine if the source field is a Picklist or a Checkbox type, and render the correct field.

Reply:
This has been changed to retrieve only the fieldMappings instead of "initializing" geformservice by calling getFormTemplate